src/data_gradients/managers/abstract_manager.py

```diff
@@ -227,6 +218,7 @@ def post_process(self, interrupted=False):
 
     def close(self):
         """Safe logging closing"""
+        self.train_data.close()
```

Comment:
only train_data? not val_data?

Reply:
ec02263d1f4b1a8c18ca31ac5d4a87c1ff54bae7